XLPandXeLaTeX.ltx

```diff
@@ -46,74 +46,69 @@ producing complex documents in beautiful formats. \TeX
+XSL‑FO\footnote{\url{https://www.w3.org/TR/xsl11}} or the \TeXML
 project\cite{lovell:1999}\footnote{\url{http://getfo.org/texml}} have also seen use to
 integrate content and compose complex documents such as textbooks
 and maintenance manuals. Requirements for composing these large, inter-linked documents
-birthed the development of tools like XMLmind\footnote{\url{https://www.xmlmind.com/xmleditor}} and
+birthed the development of tools like \XMLmind\footnote{\url{https://www.xmlmind.com/xmleditor}} and
 Xpublisher.\footnote{\url{https://www.xpublisher.com/products}} These
 can be used to compose the content within predefined \XML\ structures.
+% Why do some "XML" items have a forward slash before and after while others only before, and why do some have a forward slash and curlley brackets "{}" followin the text?
+XLingPaper\cite{blackandblack:2012},\cite{black:2009},\cite{black:2017} seeks to provide a constrained enviroment in which authors of complex works dealing with language descriptions and linguistic analysese can focus on content structure independently from the styling requirments of publishers. The software has a growing number of users who have succesfully typeset complex documents including: Master Theses\cite{Wood:2012},\cite{lamicela:2020},\cite{paterson:2021}, Doctorial Dissertations\cite{ebarb:2014},\cite{rasmussen:2018}, Textbooks\cite{marlett:2019}, Linguistic Grammars\cite{buck:2018}, Journal Articles\cite{brownie:2013}, and Bi-lingual Software Documentation Manuals\cite{BeadleandLee:2020a},\cite{BeadleandLee:2020b}.
 
-XLingPaper\cite{blackandblack:2012},\cite{black:2009},\cite{black:2017} is a plug-in to the XMLmind XML Editor. XLingPaper\footnote{\url{https://software.sil.org/xlingpaper}} benefits from XMLmind's Java-based implementation, which allows it to be used
-on MacOS, Windows, and Linux. XLingPaper, via a DTD, defines several
+XLingpaper\footnote{\url{https://software.sil.org/xlingpaper}} is a plug-in to the \XMLmind \XML Editor. XLingPaper benefits from \XMLmind's Java-based implementation, which allows it to be used on MacOS, Windows, and Linux. XLingPaper, via a DTD, defines several
 document classes (articles, books, chapters, etc., as illustrated in Figure~1),
 in each case providing document layout sections (paragraphs, examples, endnotes,
-etc.). By working within the user-interface of XMLmind, formatting
+etc.). By working within the user-interface of \XMLmind, formatting
 errors are reduced because users are constrained on where in the
 document flow they can introduce block and line level document elements.
-That is, authors cannot input \XeLaTeX\ code directly into the document and the introduction of layout sections within the document flow is constrained via the DTD.
+That is, first, authors cannot input \XeLaTeX\ code directly into the document and second, the introduction of layout sections within the document flow is constrained via the DTD.
```

Comment:
Nice!

Reply:
What does the second`\` do? in the `\XeLaTeX\ `?

Reply:
I had to go look this up to remember.  Any command word/macro like \TeX that begins with a backslash can be ended with something like "{}" or "\ ".  The latter is called a control space and makes sure there is a space after the macro.  The "{}" will not add a space.